config/routes.php

```diff
@@ -1,4 +1,4 @@
 <?php
-use Cake\Routing\Router;
-
-Router::extensions(['pdf']);
```

Comment:
Won't removing this cause BC breakage anyway? I'm not sure whether this can go in master as it will require to this to their own config. It might be reasonable to remove the file in that case.